cips/cip-draft_ica.md

```diff
@@ -0,0 +1,233 @@
+
+It’s necessary to implement liquid staking in a way that’s as safe and aligned as possible, since it’s close to the metal of PoS networks. LSPs can accumulate a large share of stake and impact the decentralization of the network, depending on how validators are selected. Given the high market demand for liquid staked TIA and impact LSPs can have on network decentralization, Celestia’s LSPs should align with Celestia’s core values: decentralization, trust minimization, and community governance. The current market offerings for liquid staking, while functional, fall short of these core principles by adding additional layers of trust and centralization risk (~$10M has been liquid staked on Celestia so far, custodied by a multisig). Multisig custody is suboptimal for a few reasons: custodians have arbitrary protocol ugprade powers, key management is complex ($1B+ has been stolen from multisig protocols due to key compromises), and custodians aren’t slashable. However, the main risk is that the market doesn’t care; recently defi projects using multisig custody have attracted over $1B.
+
+By enabling ICA on an accelerated timeline, Celestia can enable battle-tested protocols like Stride to provide more decentralized, trust-minimized liquid staking services that are more credibly governed by the Celestia community.
+
+## Specification
+
+For the full technical specification of both the host and controller module, see the [ICS-27 spec](https://github.com/cosmos/ibc/tree/main/spec/app/ics-027-interchain-accounts) in the ibc protocol repository.
+
+
+### Definitions 
+
+- `Host Chain`: The chain where the interchain account is registered. The host chain listens for IBC packets from a controller chain which contain instructions (e.g. cosmos SDK messages) that the interchain account will execute.
+- `Controller Chain`: The chain registering and controlling an account on a host chain. The controller chain sends IBC packets to the host chain to control the account.
+- `Interchain Account`: An account on a host chain. An interchain account has all the capabilities of a normal account. However, rather than signing transactions with a private key, a controller chain will send IBC packets to the host chain which signals what transactions the interchain account must execute. 
+- `Interchain Account Owner`: An account on the controller chain. Every interchain account on a host chain has a respective owner account on the controller chain. 
+
+The IBC handler interface & IBC relayer module interface are as defined in [ICS-25](../../core/ics-025-handler-interface) and [ICS-26](../../core/ics-026-routing-module), respectively.
+
+### General design 
+
+A chain can utilize one or both parts of the interchain accounts protocol (*controlling* and *hosting*). A controller chain that registers accounts on other host chains (that support interchain accounts) does not necessarily have to allow other controller chains to register accounts on its chain, and vice versa. 
```

Comment:
[question] does this proposal recommend enabling one or both parts?
1. The controlling portion of interchain accounts
2. The hosting portion of interchain accounts
3. The controlling and hosting portion of interchain accounts

I think it's either 2 or 3 but think it could be clarified.

Reply:
Resolving b/c moved to forum post: https://forum.celestia.org/t/moving-toward-safer-and-more-aligned-tia-liquid-staking/1422/15?u=rootulp

Reply:
Just 2! Added a sentence to help clarify [[here](https://github.com/celestiaorg/CIPs/pull/39/commits/2d9148935c80ccbb9d236a3f7c00e94de76ea254#diff-44ed92b035a9afbbf3dde6d410353a4bbd872168949a32a73a34d19aee5db806R30)]